.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Some of these files are from visual studio, and should be removed from the branch